--- ddccontrol-0.4.2/src/gnome-ddcc-applet/Makefile.am.orig	2006-06-15 18:45:19.000000000 +0200
+++ ddccontrol-0.4.2/src/gnome-ddcc-applet/Makefile.am	2006-08-24 23:46:30.204199000 +0200
@@ -9,8 +9,8 @@
 pkglib_PROGRAMS = ddcc-applet
 ddcc_applet_SOURCES = ddcc-applet.c ddcc-applet.h
 
-ddcc_applet_LDADD = $(GNOME_LDFLAGS) $(DDCC_LDADD)
-AM_CFLAGS = $(GNOME_CFLAGS) '-DPKGDATADIR="$(pkgdatadir)"'
+ddcc_applet_LDADD = $(GNOME_APPLET_LDFLAGS) $(DDCC_LDADD)
+AM_CFLAGS = $(GNOME_APPLET_CFLAGS) '-DPKGDATADIR="$(pkgdatadir)"'
 
 pkgdata_DATA = GNOME_ddcc-applet.xml
 
--- ddccontrol-0.4.2/configure.ac.orig	2006-07-27 22:45:07.000000000 +0200
+++ ddccontrol-0.4.2/configure.ac	2006-08-24 23:47:16.771109250 +0200
@@ -150,8 +150,8 @@
 if test x$support_gnome_applet = xyes; then
 	if pkg-config --atleast-version=2.10 libpanelapplet-2.0 ; then
 		GNOME_APPLET="gnome-ddcc-applet"
-	    GNOME_LDFLAGS="$LIBXML2_LDFLAGS `pkg-config --libs gtk+-2.0 gthread-2.0 libpanelapplet-2.0`"
-	    GNOME_CFLAGS="$LIBXML2_CFLAGS `pkg-config --cflags gtk+-2.0 gthread-2.0 libpanelapplet-2.0`"
+	    GNOME_APPLET_LDFLAGS="$LIBXML2_LDFLAGS `pkg-config --libs gtk+-2.0 gthread-2.0 libpanelapplet-2.0`"
+	    GNOME_APPLET_CFLAGS="$LIBXML2_CFLAGS `pkg-config --cflags gtk+-2.0 gthread-2.0 libpanelapplet-2.0`"
 	fi
 fi
 
@@ -160,6 +160,8 @@
 AC_SUBST([GNOME_CFLAGS])
 AC_SUBST([GDDCCONTROL])
 AC_SUBST([GNOME_APPLET])
+AC_SUBST([GNOME_APPLET_LDFLAGS])
+AC_SUBST([GNOME_APPLET_CFLAGS])
 
 # Doc check
 support_doc=no
