diff -urN ddccontrol-0.4.2.orig/configure.ac ddccontrol-0.4.2/configure.ac
--- ddccontrol-0.4.2.orig/configure.ac	2007-01-04 00:08:09.945690046 +0100
+++ ddccontrol-0.4.2/configure.ac	2007-01-04 00:10:59.643360565 +0100
@@ -101,7 +101,7 @@
 DDCPCI=
 if test x$support_ddcpci = xyes; then
    AC_CHECK_HEADERS([pci/pci.h], [], [AC_MSG_ERROR([PCI utils headers not found, please install pci-utils.], [1])], [])
-   AC_CHECK_LIB([pci], [pci_alloc], [], [AC_MSG_ERROR([PCI utils library not found, please install pci-utils.], [1])])
+   AC_CHECK_LIB([pci], [pci_alloc], [], [AC_MSG_ERROR([PCI utils library not found, please install pci-utils.], [1])], [-lz])
    DDCPCI=ddcpci
    AC_DEFINE_UNQUOTED(HAVE_DDCPCI, 1, [Define if ddccontrol is built with ddcpci support.])
 fi
diff -urN ddccontrol-0.4.2.orig/src/ddcpci/Makefile.am ddccontrol-0.4.2/src/ddcpci/Makefile.am
--- ddccontrol-0.4.2.orig/src/ddcpci/Makefile.am	2007-01-04 00:08:09.929689134 +0100
+++ ddccontrol-0.4.2/src/ddcpci/Makefile.am	2007-01-04 00:20:26.227648363 +0100
@@ -1,7 +1,7 @@
 INCLUDES = -I../lib  -DLOCALEDIR=\"$(localedir)\"
 
 bin_PROGRAMS = ddcpci
-ddcpci_LDFLAGS = -Wl,-z,now
+ddcpci_LDFLAGS = -Wl,-z,now,-lz
 ddcpci_SOURCES = main.c nvidia.c radeon.c i2c-algo-bit.c intel810.c ddcpci.h i2c-algo-bit.h via.c sis.c
 
 install-exec-hook:
