--- ddccontrol-1.0.3/src/gddccontrol/gddccontrol.desktop.in~	2024-01-31 09:38:39.000000000 +0000
+++ ddccontrol-1.0.3/src/gddccontrol/gddccontrol.desktop.in	2025-08-19 23:00:39.417976924 +0000
@@ -2,8 +2,10 @@
 _Name=DDC Control
 _GenericName=Monitor Settings
 _Comment=Set your monitor preferences (contrast, brightness,...) and manage your profiles
+GenericName[pl]=Ustawienia monitora
+Comment[pl]=Ustaw parametry pracy monitora (kontrast, jaskrawość, ...) i zarządzaj profilami ustawień
 Exec=gddccontrol
-Icon=gddccontrol
+Icon=gddccontrol.png
 Terminal=false
 Type=Application
 Categories=GTK;Settings;HardwareSettings;
